aptos-move/framework/src/natives/state_storage_context.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_types::vm_status::StatusCode;
+use better_any::{Tid, TidAble};
+use move_deps::move_binary_format::errors::PartialVMError;
+use move_deps::move_vm_types::pop_arg;
+use move_deps::move_vm_types::{
+    loaded_data::runtime_types::Type,
+    natives::function::NativeResult,
+    values::{Struct, Value},
+};
+use move_deps::{
+    move_binary_format::errors::PartialVMResult,
+    move_vm_runtime::native_functions::{NativeContext, NativeFunction},
+};
+use smallvec::smallvec;
+use std::collections::VecDeque;
+use std::sync::Arc;
+
+/// FIXME(aldenhu): doc
+pub struct StateStorageUsage {
+    items: usize,
+    bytes: usize,
+}
+
+/// FIXME(aldenhu): doc
+/// A table resolver which needs to be provided by the environment. This allows to lookup
+/// data in remote storage, as well as retrieve cost of table operations.
```

Comment:
Seems like this is old documentation copied from the table extension impl.

Reply:
yeah will fix all the `FIXME(aldenhu)`s before publishing it